src/frontend/src/flows/manage/settingsDropdown.ts

```diff
@@ -1,22 +1,42 @@
 import { dropdownIcon } from "$src/components/icons";
+import { handleKeyPress } from "$src/utils/keyboard.utils";
+import { Chan } from "$src/utils/utils";
 import { html, TemplateResult } from "lit-html";
+import { asyncReplace } from "lit-html/directives/async-replace.js";
 
-export const settingsDropdown = ({
-  alias,
-  id,
-  settings,
-}: {
+interface SettingsDropdown {
   alias: string;
   id: string;
   settings: { fn: () => void; caption: string; action: string }[];
-}): TemplateResult => {
+}
+
+const mkChasm = ({ alias, id, settings }: SettingsDropdown): TemplateResult => {
+  /* Toggle the chasm open/closed */
+  const ariaExpanded = new Chan(false);
+  const chasmToggle = () => ariaExpanded.send(!ariaExpanded.latest);
+
+  const backdrop = ariaExpanded.map((expanded) =>
+    expanded
+      ? html`<div
+          class="c-dropdown__backdrop"
+          role="button"
+          tabindex="-1"
+          @click=${() => chasmToggle()}
```

Comment:
This should only ever close the dropdown, correct? Would `ariaExpanded.send(false)` make sense?